code/game/gamemodes/handofgod/items.dm

```diff
@@ -201,6 +195,29 @@
 		user << "A powerful staff capable of changing the allegiance of god/demigod structures."
 
 
+
+/obj/item/weapon/godstaff/attack_self(mob/living/carbon/user)
+	if(god && !god.is_handofgod_myprophet(user))
+		user << "<span class='danger'>YOU ARE NOT THE CHOSEN ONE!</span>"
+		return
```

Comment:
when one of the god is killed, god will return FALSE, so anyone using the staff will get "Your connection to your diety isn't strong enough!" whether or not they are a believer.